generator/src/test/java/com/scottlogic/deg/generator/generation/FieldSpecValueGeneratorTests.java

```diff
@@ -30,7 +31,7 @@ void generate_fieldSpecContainsSingleMustContainRestriction_returnsValuesWithVal
             .withSetRestrictions(
                 new SetRestrictions(
                     new HashSet<>(
-                        Arrays.asList(1, 10)
+                        Arrays.asList(1, 5, 10)
```

Comment:
This defeats the purpose of the test, the test originally was designed to ensure `5` was emitted even though its **NOT** in the set restriction.

Reply:
We believe the system was previously bugged and the tests were asserting incorrect behaviour.
It was testing that with a root node with a set of [1, 10] and a decision with a set of [5] would produce [1, 5, 10]
However the [5] is contradictory to the [1, 10], so this behaviour was incorrect.

we are now testing  that with a root node with a set of [1, 5, 10] and a decision with a set of [5] would produce [1, 5, 10]
which is now more of a "does it duplicate 5" test

Reply:
Understood.

- Does the test add any value, does it also describe that it is trying to assert (i.e. duplication now)?
- Should it be updated to use greaterThan/lessThan so the test is corrected rather than updated. Are there other tests that do cover this scenario

Reply:
I think a greaterThan/lessThan case is already tested. Also, I believe it does add value, but is misleading, so will change the name of the test